include/liblwm2m.h

```diff
@@ -124,6 +124,11 @@ time_t lwm2m_gettime(void);
 void lwm2m_printf(const char * format, ...);
 #endif
 
+#ifdef LWM2M_WITH_COAP_LOGS
+// Same usage as C89 printf()
```

Comment:
I think C89 is too specific (will also work with C99, C11, etc.)

```suggestion
// Same usage as printf()
```

Reply:
I just copied this from the comment above `lwm2m_printf(...)`, anyway no problem, I'll fix this.